js/IdentityProxyDeploy.js

```diff
@@ -1,15 +1,87 @@
-const solc = require('solc')
+const swarmhash = require('swarmhash')
 const abi = require('ethereumjs-abi')
 const keccak256 = require('js-sha3').keccak256
 const assert = require('assert')
 
-function getMappingSstore(slotNumber, keyType, key, value) {
+function Generator() {
```

Comment:
rename that, cause `Generator` has other meaning (ES6 generators)

Reply:
I renamed `Generator` to `Assembler`